docs/General/Manuals/client-manual.rst

```diff
@@ -1403,10 +1403,53 @@ and Global Warming chance give a good indication of what the icon looks like.
 Messages
 ^^^^^^^^
 
+The :guilabel:`Messages` icon is a button on the `Top Function Bar`_ that is used to show or hide the messages
+dialog window. Figure 40 below shows an example of the :guilabel:`Messages` icon button.
```

Comment:
Consider rephrasing to "The Messages button on the Top Function Bar is used to toggle the message log widget. Figure 40 below shows an example of the Messages button."